cpp/include/cudf/io/parquet.hpp

```diff
@@ -58,9 +58,9 @@ class parquet_reader_options {
   // List of individual row groups to read (ignored if empty)
   std::vector<std::vector<size_type>> _row_groups;
   // Number of rows to skip from the start
-  size_type _skip_rows = 0;
-  // Number of rows to read; -1 is all
-  size_type _num_rows = -1;
+  int64_t _skip_rows = 0;
```

Comment:
Why is this an `int64_t` when the orc uses `uint64_t`? Sticking to signed in both cases is probably preferable.

Reply:
Same as with ORC, we're matching the Parquet specs, where the row count is stored as signed int64.
Unless you think this is very error-prone, I'd like us to start using the types as they are in the format specs. Having aliases for these would also go a long way to make the code more readable.